src/mocks/hgv-record.mock.ts

```diff
@@ -0,0 +1,104 @@
+import { VehicleTechRecordModel, StatusCodes, VehicleTypes, FuelTypes, VehicleConfigurations, EuVehicleCategories, approvalType, PlateReasonForIssue, BodyTypeDescription } from '../app/models/vehicle-tech-record.model';
+import { createMock } from "ts-auto-mock";
+
+export const createMockHgv = (systemNumber: number): VehicleTechRecordModel =>
```

Comment:
pedantic but the formatting seems off here. Try running the lint script with `--fix`

Reply:
What don't you like? My formatter isn't changing it?